bindings/src/main/java/io/pravega/storage/extendeds3/ExtendedS3StorageConfig.java

```diff
@@ -33,6 +33,7 @@
     public static final Property<Boolean> USENONEMATCH = Property.named("useNoneMatch", false);
 
     private static final String COMPONENT_CODE = "extendeds3";
+    private static final String PATH_SEPARATOR = "/";
```

Comment:
This should go into `StreamSegmentNameUtils.java` and you should update all references to the string `"/"` in there too.

Reply:
StreamSegmentNameUtils is in ':shared:protocol' project which has mostly protocol related stuff. Today 'bindings' project does not depend on it and it doesn't make sense to bring in that dependency just for this change.